examples/replace-component/src/actions.js

```diff
@@ -0,0 +1,6 @@
+import {createAction} from 'redux-actions';
+
+export const setMapConfig = createAction(
+  'SET_MAP_CONFIG',
+  payload => payload
```

Comment:
Does `createAction` require a second arg if the payload formatter is the identity fn?

Reply:
certain version of redux-actions will throw error if payload creator is undefined.